_posts/2016-08-02-internship-thoughts.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "A Savas Labs Internship"
+date: 2016-08-02
+author: Ro Wang
+tags:
+summary: What it's like to learn at Savas Labs.
+---
+<img src="/assets/img/blog/internship-slave.jpg" alt="Sipress Cartoon">
+
+<span class="caption">[_Source_](https://www.cartoonbank.com/search/-/search/results?p_p_state=pop_up&_listenersearchresults_WAR_searchportlet_struts.portlet.action=%2Fview%2FshowDetail&_listenersearchresults_WAR_searchportlet_pageNumber=1&_listenersearchresults_WAR_searchportlet_search=feature%3Acncartoons+intern&_listenersearchresults_WAR_searchportlet_match=a&_listenersearchresults_WAR_searchportlet_sort=d&_listenersearchresults_WAR_searchportlet_byline=Sipress&_listenersearchresults_WAR_searchportlet_docdaterng=2004&_listenersearchresults_WAR_searchportlet_searchFeatures=cncartoons&_listenersearchresults_WAR_searchportlet_tagId=cncartoons022239).</span>
+
+If you've ever contemplated a drastic career change, then you've probably thought about what it'd be like to start at the beginning again at some point.
+
+In this post, I'd like to share my 3 month internship experience for anyone who's considering making the switch to Web Development, or who may be curious about the learning environment here at Savas.
+
+## Visiting the Team
+
+Since I’m located [2,845 miles](https://www.google.com/maps/dir/Portland,+OR/Durham,+NC/@39.4988195,-118.778336,4z/data=!4m13!4m12!1m5!1m1!1s0x54950b0b7da97427:0x1c36b9e6f6d18591!2m2!1d-122.6764816!2d45.5230622!1m5!1m1!1s0x89ace471120f66f1:0xe17ada898a46f27e!2m2!1d-78.898619!2d35.9940329) from Savas Labs, one of my biggest concerns starting out as a newbie was the added challenge of telecommuting. To get me up and running as fast as possible, [Chris](http://savaslabs.com/team/chris-russo/) and I decided I’d fly to Durham to meet the team and work in the office together for my first week.
```

Comment:
Since the time your internship started,  we became majority non Durhamites so maybe rephrase this as Savas Labs HQ?